src/main/java/org/jabref/gui/entryeditor/CommentsTab.java

```diff
@@ -140,7 +142,7 @@ protected void setupPanel(BibEntry entry, boolean compressed) {
             boolean isStandardBibtexComment = field == StandardField.COMMENT;
             boolean isDefaultOwnerComment = field.equals(userSpecificCommentField);
             boolean shouldBeEnabled = isStandardBibtexComment || isDefaultOwnerComment;
-            editor.getNode().setDisable(!shouldBeEnabled);
+            ((MarkdownEditor) editor.getNode()).setEditable(shouldBeEnabled);
```

Comment:
Do this cast on line 140, where you do `getValue()`.

Reply:
> Do this cast on line 140, where you do `getValue()`.

Done！